pkgs/http_client_conformance_tests/lib/src/response_body_streamed_test.dart

```diff
@@ -77,5 +77,63 @@ void testResponseBodyStreamed(Client client,
       });
       await cancelled.future;
     });
+
```

Comment:
[optional] Would you mind adding some tests to check that stream pause and resume works?

Reply:
I have added a few tests ensuring pause and resume work. Let me know if you were looking for more. To properly test this, one would probably have to check that pausing a stream on the client eventually also pauses the stream on the server due to flow control. But that seems like a very tricky test to write.